features/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/include/psa/initial_attestation.h

```diff
@@ -46,7 +46,7 @@ extern "C" {
  * attestation service. Used to configure buffers for services that verify the
  * produced tokens.
  */
-#define PSA_INITIAL_ATTEST_MAX_TOKEN_SIZE (0x400)
+#define PSA_INITIAL_ATTEST_MAX_TOKEN_SIZE (0x500)
```

Comment:
Does this need to change? If so, why? Unless this change is upstream, it's likely to get overwritten with the next TF-M upgrade and v1.1 just came out).

Reply:
We will upstream it later.